nexmark-flink/src/main/java/com/github/nexmark/flink/QueryRunner.java

```diff
@@ -65,6 +65,16 @@ public JobBenchmarkMetric run() {
 			System.out.println("Start to run query " + queryName + " with workload " + workload.getSummaryString());
 			LOG.info("==================================================================");
 			LOG.info("Start to run query " + queryName + " with workload " + workload.getSummaryString());
+			if (!"insert_kafka".equals(queryName) && (workload.getWarmupEvents() > 0L || workload.getWarmupMills() > 0L)) {
```

Comment:
Should we also consider `workload.getWarmupTps() > 0` ?

Reply:
Good point, I will add a sanity check here.
And since the configuration of tps and events is not valid when using kafka source, I will also check whether the ```workload.getKafkaServers()``` is ```null```